src/EventSubscriber/UserLocaleSubscriber.php

```diff
@@ -38,6 +38,10 @@ public function __construct(SessionInterface $session)
 
     public function onInteractiveLogin(InteractiveLoginEvent $event)
     {
+        if (!$this->session->isStarted()) {
```

Comment:
```suggestion
        if (!$event->getRequest()->hasPreviousSession()) {
```

Since the `InteractiveLoginEvent` has the `request` object, what about removing the `$session` attribute of this class and retrieving the session from the request? I don't remember exactly, but sometimes simple accessing the session, starts it.

Reply:
Thank you @franmomu. Your suggestion is addressed.